recipes/fortnet/build.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+set -ex
+
+if [ "${mpi}" != "nompi" ]; then
+  MPI=ON
+else
+  MPI=OFF
+fi
+
+cmake_options=(
+   ${CMAKE_ARGS}
+   "-DLAPACK_LIBRARY='lapack;blas'"
+   "-DWITH_SOCKETS=ON"
+   "-DWITH_MPI=${MPI}"
+   "-DBUILD_SHARED_LIBS=ON"
+   "-GNinja"
+   ${cmake_mpi_options[@]}
+   ..
+)
```

Comment:
```suggestion
cmake_options=(
   ${CMAKE_ARGS}
   "-DCMAKE_BUILD_TYPE=Release"
   "-DLAPACK_LIBRARY='lapack;blas'"
   "-DWITH_SOCKETS=ON"
   "-DWITH_MPI=${MPI}"
   "-DBUILD_SHARED_LIBS=ON"
   "-GNinja"
   ${cmake_mpi_options[@]}
   ..
)
```
We want Release, [not ReleasewithDebInfo.](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=598957&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=2071)

Reply:
I see, it's now compiled in Release mode.